cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -106,110 +88,215 @@ bool FliImpl::isValueBoolean(mtiTypeIdT type)
     return false;
 }
 
-bool FliImpl::isTypeValue(int type)
+bool fli_is_signal(void *hdl)
+{
+    return (acc_fetch_type(hdl) == accSignal || acc_fetch_fulltype(hdl) == accAliasSignal);
```

Comment:
An aside - where is the definition of `acc_fetch_type`? I'm struggling to even find a signature for that function online, let alone documentation.

Reply:
I believe acc_fetch_type() is part of the PLI interface, but not sure on that, no real documentation.  I believe @stuarthodgson was the first to add using that function as it is the only way to get what type handle is with more detail, i.e. constant, generic, generate loop, etc.  I doubt you will find anything on the web about the interface.

Reply:
Is there even a copy of the header file I can obtain somewhere?

Reply:
I am not comfortable distributing it since it is mentor's file.  So I would say unless you have Modelsim or Questa, probably not.

Reply:
That's fair. Can you tell me the name of the header it's defined in, so that I can try and source it from elsewhere?

Reply:
acc_user.h and acc_vhdl.h